Lib/concurrent/futures/process.py

```diff
@@ -125,6 +125,7 @@ class _ExceptionWithTraceback:
     def __init__(self, exc, tb):
         tb = ''.join(format_exception(type(exc), exc, tb))
         self.exc = exc
+        self.exc.__traceback__ = None
```

Comment:
Is there a way to test this? If not, there should be a comment here explaining why we clear the traceback.

Reply:
Unfortunately anything small enough for a unit test does not fail with the current codebase. Comment added.